index.Rmd

```diff
@@ -8,7 +8,7 @@ output:
 # Introduction
 
 ## Background
-Food Insecurity is a serious issue that affects 10.5% of America as of 2019, according to NPR (2020). Additionally, due to systemic racism, Black and Hispanic Americans are twice as likely to face food insecurity compared to their white counterparts. Particularly, for Erie County, Feeding America (2020) states that 12% of residents are food insecure (110,390 individuals). Community efforts to support Buffalo and Erie County residents have taken the form of urban agriculture and community fridges. Besides providing food provisioning, studies have also shown that these efforts have many other social perks such as recreational benefits and a decrease in crime (Hess and Winner, 2007). The goal of this website is to show how these community efforts have improved food security over time. Demographic data will also be looked at. However, there are some limitations to this. There wasn't information on when each garden was founded, so some have no start dates. The other limitation is these maps right now can only show how food security may have improved and not the other benefits that were previously mentioned. Also, the COVID-19 pandemic did generally worsen food insecurity. Nevertheless, the idea is to show the importance of these community efforts.
```

Comment:
You should probably cite where you got the data for the line "Additionally, due to systemic racism, Black and Hispanic Americans are twice as likely to face food insecurity compared to their white counterparts."